app/dashboard/static/js/app/view-builds-job-branch-kernel.2017.7.1.js

```diff
@@ -326,6 +329,13 @@ require([
                 archLabelNode.appendChild(document.createTextNode(arch));
             }
 
+            hNode.insertAdjacentHTML(
+                'beforeend', '&nbsp;&dash;&nbsp;');
+            compilerLabelNode = hNode.appendChild(
+                document.createElement('span'));
+            compilerLabelNode.setAttribute('class', 'compiler-label');
+            compilerLabelNode.appendChild(document.createTextNode(compiler));
+            
```

Comment:
I think there is some extra white space on this line even though it doesn't appear on GitHub.

Reply:
I checked it here and pushed again.